internal/proxy/hook/authz/authz_test.go

```diff
@@ -109,8 +95,6 @@ func TestCreateResources(t *testing.T) {
 
 	for _, tt := range table {
 		t.Run(tt.title, func(t *testing.T) {
```

Comment:
The right way to fix this test for it to be run in parallel would be to redeclare `tt` inside the for loop scope:

```go
for _, tt := range table {
	tt := tt
	t.Run(tt.title, func(t *testing.T) {
// ...
```